Sprint-1/exercises/random.js

```diff
@@ -4,6 +4,31 @@ const maximum = 100;
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
 // In this exercise, you will need to work out what num represents?
+  //Math.floor(Math.random() * (maximum - minimum + 1)) + minimum
+  //Math.floor(Math.random() * (100 - 1 + 1)) + 1
+  //Math.floor(Math.random() * (100)) + 1
+  //Math.floor(0.1812 * (100)) + 1
+  //Math.floor(18.12) + 1
+  //18 + 1
+  //19=this could be change as the Math.random() function could returns any random value
 // Try breaking down the expression and using documentation to explain what it means
+  //Math.random()= returns a random value between 0 and 1
```

Comment:
The phrase "between 0 and 1" alone is not precise enough in program specification because 
it does not state clearly whether 0 and 1 are included in the range.

May I suggest asking ChatGPT how to precisely describe a range of numbers from 0 to 1 
that include/exclude 0/1, and then use what you learn to describe the return value of 
`Math.random()`?